README.md

```diff
@@ -18,10 +18,198 @@ Or install it yourself as:
+  * path
+  * regex
+  * timeout (defaults to 5s)
+
+- `Dependency::Script`: Checks whether the output of a command matches a string or regex.
+  * command
+  * timeout (defaults to 5s)
+
+- `Metric::ConstantMetric`: A dummy metric that always reports a constant.
+  * weight (0-100)
+
+- `Metric::CPULoad`: Normalizes CPU load to a value between 0-100 and inverts it, so higher numbers mean less load and lower numbers mean more. Final health is weighted against other checks by `:weight`.
+  * weight (0-100)
+
+- `Metric::InternetHealth`: Checks connectivity across a set of hosts and computes a weight based on how many are reachable. Helpful if you want to check outbound connectivity through multiple ISPs.
+  * weight (0-100)
+  * hosts
+  * timeout (defaults to 5s)
+
+- `Metric::Script`: Runs a script to obtain a health from 0-100. This is helpful for customized metrics.
+  * command
+  * weight (0-100)
+  * timeout (defaults to 5s)
+
+- `Metric::BigBrotherService`: Used in conjunction with [Big Brother](https://github.com/braintree/big_brother), reports health based on the overall health of another load balanced service.
+  * service
+
+### HAProxy agent check configuration
+
+Litmus paper can also report health checks in HAProxy agent check format. The agent check functionality takes the health data from a service health check, and exposes it on a different port in the format HAProxy expects.
+
+There are no configuration files for the agent check, since all options are specified on the command line.
+
+```
+Usage: litmus-agent-check [options]
+    -s, --service SERVICE:PORT,...   agent-check service to port mappings
+    -c, --config CONFIG              Path to litmus paper config file
+    -p, --pid-file PID_FILE          Where to write the pid
+    -w, --workers WORKERS            Number of worker processes
+    -D, --daemonize                  Daemonize the process
+    -h, --help                       Help text
+```
+
+The service:port argument means that the server will expose the data from the litmus check for `service` on `port` in HAProxy agent check format. For example, if you wanted to serve status information about `myapp` on port `8080`, and already had a service config for it, you'd pass `-s myapp:8080`.
+
+On the HAProxy server, add `agent-check agent-port 8080 agent-inter <seconds>s` to the config line for each server listed for that backend. This tells HAProxy to query port 8080 on the backend every `<seconds>` seconds for health information.
+
+### REST API
+
+- /
+  * GET: Returns table with status of each service.
+
+		Litmus Paper 1.1.1
+
+		 Service    │ Reported │ Measured │ Health
+		 Name       │ Health   │ Health   │ Forced?
+		────────────┴──────────┴──────────┴─────────
+		 myapp            0        100      Yes, Reason: testing
+		 myotherapp      72         72      No
+
+- /down
+  * POST: Creates a global force down. Parameters: reason => reason for the force down.
```

Comment:
params - url params? json body? que?

Reply:
I think these are POST request json body params - will confirm and clarify.